spec/takeaway_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'takeaway'
+
+RSpec.describe Takeaway do
+
+  subject(:takeaway)  {described_class.new}
+
+    it 'sees a list of dishes with prices' do
+      expect(takeaway.display_menu).to eq ({"chicken" =>3,
+     "rice" =>1,
+     "pizza" =>6,
+     "chips" =>2 })
+    end
+
+    it {is_expected.to respond_to(:select_dishes).with(2).arguments}
+
+    it 'grows the basket after selection' do
+      expect{takeaway.select_dishes('chicken', 4)}.to change{takeaway.basket.size}.by 1
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)